tools/util.py

```diff
@@ -171,11 +173,11 @@ def find_exts(directories, extensions, skip=None):
         skip = []
     assert isinstance(directories, list)
     assert isinstance(extensions, list)
-    skip = [os.path.normpath(i) for i in skip]
+    skip = [os.path.abspath(i) for i in skip]
     matches = []
     for directory in directories:
         for root, dirnames, filenames in os.walk(directory):
-            if root in skip:
+            if os.path.abspath(root) in skip:
```

Comment:
This also fixes the issue with not being able to use subdirectories in `directories` on Windows, because the not normalized version would be compared with the normalized version otherwise. 👍